builder-image-aws/Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM alpine:3.9
 
+LABEL org.opencontainers.image.source https://github.com/liatrio/builder-images
```

Comment:
Was looking at alternatively using the label flag in the skaffold build but that's probably only for kaniko builds.  No problem, this option works well.